lib/cisco_node_utils/bgp.rb

```diff
@@ -311,6 +311,22 @@ def default_confederation_id
       config_get_default('bgp', 'confederation_id')
     end
 
+    # Enforce First As (Getter/Setter/Default)
+    def enforce_first_as
+      match = config_get('bgp', 'enforce_first_as', @get_args)
+      match.nil? ? false : default_enforce_first_as
```

Comment:
I spoke with Andi about this offline. The other boolean getters in this file are using the same method so I told him to just leave it as is and we'll fix these all in a separate commit.